pandera/schemas.py

```diff
@@ -471,6 +471,23 @@ def name(self) -> Union[str, None]:
         """Get SeriesSchema name."""
         return self._name
 
+    @property
+    def pandas_dtype(self) -> Union[
+            str,
+            dtypes.PandasDtype,
+            dtypes.PandasExtensionType]:
+        """Get the pandas dtype"""
+        return self._pandas_dtype
+
+    @pandas_dtype.setter
+    def pandas_dtype(self, value: Union[
+            str,
+            dtypes.PandasDtype,
+            dtypes.PandasExtensionType]) -> None:
+        """Set the pandas dtype"""
+        #TODO do we need specfifi checks or actions for consistency here?
```

Comment:
yeah, perhaps it would be good to check that `self.dtype` (returns string alias for pandas dtype) returns string or None, raising error if the provided `value` is invalid. Unit tests to reflect this would also be great 👍🏽

Reply:
(FYI the CI is complaining about this TODO comment)